src/components/LoanCollections/KivaClassicLoanGrid.vue

```diff
@@ -0,0 +1,246 @@
+<template>
```

Comment:
This seems like a full replica with minor additions to https://github.com/kiva/ui/blob/main/src/components/LoanCollections/KivaClassicLoanCarousel.vue

Can we just pass a prop and conditionally alter the 2 additions you've made here?